utilities/scripts/Get-ModulesAsMarkdownTable.ps1

```diff
@@ -0,0 +1,164 @@
+<#
```

Comment:
It's planned to have this running within Workflows and get it auto-generated every run.
Not sure if it's worth it to bring in utilities into the root or keep them under .github/workflows/scripts.